app/src/controllers/ChannelController.js

```diff
@@ -0,0 +1,224 @@
+const connext = require('@connext/client')
+
+class ChannelController {
+  /**
+   * @constructor
+   * @param {Object} opts
+   */
+  constructor(opts) {
+    this.networkController = opts.networkController
+    this.keyringController = opts.keyringController
+    this.store = opts.store
+  }
+
+  /**
+   * Helper method for initializing the connext client.
+   *
+   * This should be called any time on app startup, or before
+   * using any channel methods.
+   *
+   * @returns Promise<void>
+   */
+  initializeConnext() {
+    const { ETH_PROVIDER_URL, NODE_URL, LOG_LEVEL } = process.env
+
+    const xpub = this.keyringController.getChannelXPub()
+    const keyGen = this.keyringController.getChannelKeyGen()
+
+    const connectOpts = {
+      xpub,
+      keyGen,
+      ethProviderUrl: ETH_PROVIDER_URL || 'https://rinkeby.indra.connext.network/api/ethprovider', // default to nodes provider
+      logLevel: LOG_LEVEL || 5, // default to log everything
```

Comment:
we have a loglevel package and please use that everywhere for all logs.
we take care of the level of logging based on build environment

Reply:
This is a hard one because we use our logger internally, would it be possible to map the log level?

Reply:
Yes it's possible.
Or, please disable all logs for production. (process.env.VUE_APP_TORUS_BUILD_ENV==='production')